knowlg-app/package.json

```diff
@@ -13,40 +13,49 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "^14.0.0",
```

Comment:
Why angular is downgraded

Reply:
Before library was in angular 9 and portal application also was in 9 version but after migration library was not supporting so did these changes for compatible.